godtools/Views/TractElements/TractTextContentProperties.swift

```diff
@@ -15,11 +15,11 @@ class TractTextContentProperties: TractProperties {
     var i18nId: String = ""
     var textAlign: NSTextAlignment = .left
     // textColor
-    var textScale: CGFloat = 1.0
+    // textScale
     var value: String = ""
     
     override func defineProperties() {
-        self.properties = ["i18nId", "textScale", "value"]
```

Comment:
No, the text-scale will always be used, but now it is defined as a general property, not just for texts, because it can be set on many places